packages/aserto/src/aserto/client/directory/v2/__init__.py

```diff
@@ -15,27 +15,108 @@
 
 import aserto.client.directory as directory
 import aserto.client.directory.v2.helpers as helpers
-from aserto.client.directory import NotFoundError
+from aserto.client.directory import ConfigError, NotFoundError
 from aserto.client.directory.v2.helpers import ObjectIdentifier, RelationResponse
 
 
+def build_grpc_channel(address: str, default_address: str, ca_cert_path: str) -> Optional[grpc.Channel]:
+    if address == "" and default_address == "":
+        return None
+        
+    return grpc.secure_channel(
+        target=address or default_address, 
+        credentials=directory.channel_credentials(cert=ca_cert_path),
+    )
+
 class Directory:
     def __init__(
         self,
         *,
-        address: str,
         api_key: str = "",
         tenant_id: str = "",
         ca_cert_path: str = "",
+        address: str = "",
+        reader_address: str = "",
+        writer_address: str = "",
+        importer_address: str = "",
+        exporter_address: str = "",
```

Comment:
The model_address is missing from the list.

Reply:
This is v2 directory